### PR TITLE
Implement `.pr$DataFrame$drop_all_in_place()`

### PR DESCRIPTION
**Pull request by sorhawell**
support dropping pl::DataFRame inside RPolarsDataFrame to assist https://github.com/rpolars/r-polarssql/issues/2

No news because it is an internal method. 
If r-polarssql chooses to rely on this method, add comment to not break behavior.

### CODE REVIEW

**Review by eitsupi — APPROVED** [review]
Thanks! I think this is useful.

@etiennebacher Any thoughts?

**Comment by etiennebacher** [comment]
I'll add a few comments in a couple of hours

File: tests/testthat/test-dataframe.R [hunk 1]
```diff
@@ -1206,3 +1206,19 @@ test_that("transpose", {
...
+
+test_that("drop_all_in_place", {
+
+  # this test verifies internal function in_place drop all Series in DataFrame
+  # will not affect a fully cloned DataFrame df_clone
+
+  df_copy = df = polars::pl$DataFrame(mtcars)
+  df_clone = df$clone()
+  s = df$get_column("cyl")
+  .pr$DataFrame$drop_all_in_place(df) # we make this method and instead drop in place - level 3-5
```
[the remark this answers is not in the document]
Comment (sorhawell) [reply tests/testthat/test-dataframe.R:1218]: removed

File: src/rust/src/rdataframe/mod.rs [hunk 2]
```diff
@@ -124,6 +124,11 @@ impl DataFrame {
         LazyFrame(self.0.clone().lazy())
     }
 
+    //internal use only
```
[the remark this answers is not in the document]
Comment (sorhawell) [reply src/rust/src/rdataframe/mod.rs:127]: adding news

**Pull request merged**